local_build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+VERSION=0.0.10
+./pants -ldebug publish.jar --local=~/.m2/repository --no-dryrun --override="com.thesamet#play-pants-tool=$VERSION" src/scala/playpants/tool:play-pants-tool
+aws s3 cp --recursive ~/.m2/repository/com/thesamet/play-pants-tool/$VERSION-SNAPSHOT/ s3://aiq-artifacts/releases/com/thesamet/play-pants-tool/$VERSION-SNAPSHOT/
```

Comment:
Hmm how come we never had to do this before? I thought maybe there was some magic inside the play-pants plugin, it would build this tool at runtime itself?

Reply:
i dont think we changed the jar before?  but i dunno

Reply:
Weird, yeah I guess it doesnt make sense, you cant build a jar at runtime. So maybe it was just never picking up the changes to the jar? 🤦